src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1417,5 +1417,26 @@ internal virtual IntermediateNode CreateIRCallNode(CallNode node, IRTranslatorCo
 
             return new IRCallNode(context.GetIRContext(node), this, args);
         }
+
+        /// <summary>
+        /// Function can override this method to provide BlankHandling policy for argument.
```

Comment:
Should this have a more generic name now?

Reply:
updated.